src/types/routeMethods.spec-d.ts

```diff
@@ -376,7 +376,7 @@ test('all routes with a name can be called unless disabled', () => {
   expectTypeOf(router.routes.parent.child).toBeFunction()
   expectTypeOf(router.routes.parent2).not.toBeFunction()
   expectTypeOf(router.routes.parent2.child2).toBeFunction()
-  expectTypeOf(router.routes.parent2).not.toHaveProperty('child3')
+  expectTypeOf(router.routes.parent2.child3).toMatchTypeOf<never>()
```

Comment:
Removing the `Identity` type from `RouteMethods` gives significant performance improvements. The downside is now non public routes show up as `never`. I think we can probably fix this but for now not a high priority. Functionally and devx wise still pretty good as is.